### DISCUSSION TITLE
your_secret_name

### DISCUSSION THREAD

**Discussion by mariepreston66** [community/community#70336] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

How do you create a 

your_secret_name

to add to a forked repository, before you add the six secrets?

Help!
Peter

**Comment by Dineth-H** [comment, marked as answer]
You can create a secret name for a forked repository, before you add the six secrets as below,

1. Go to the repository settings page.
2. Click on the Secrets and variables tab.
3. Under Actions, click on the Secrets tab.
4. Click on New repository secret.
5. In the Name field, enter the name for your secret.
6. In the Secret field, enter the value for your secret.
7. Click on Add secret.

Your secret name will now be displayed in the list of secrets. You can then add the six secrets to your forked repository.

for more, read the GitHub actions documentation at [GitHub Security Guides](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions)

**Reply by ghostinhershell** [reply]
Thanks for contributing here! Cheers 🎉

**Reply by mariepreston66** [reply]
Hi Dineth,
thanks for your help
Peter
